web/.gitignore

```
@@ -32,7 +29,9 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+renderer/*.tsbuildinfo
 next-env.d.ts
+renderer/next-env.d.ts
```

Does this work to ignore files at all levels?
```suggestion
**/*.tsbuildinfo
**/next-env.d.ts
```